app/event-list/event-list.component.ts

```diff
@@ -32,8 +39,14 @@ export class EventListComponent implements OnInit {
 
     public ngOnInit(): void {
         this._sideDrawerTransition = new SlideInOnTopTransition();
+        // console.log("TESTING before:: "); 
+        // let p = this._eventServices.loadEvents<Array<IEvent>>().
+        //     then((newSessions: Array<IEvent>) => {
+        //         console.log("DO NOTHING");
+        //     });
+        // console.log("TESTING :: ", p);
```

Comment:
Probably best to delete the commented out code.

Reply:
Left it uncommented because its a working implementation but just seeing an issue with line 43 which I need to look into.